include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -216,4 +216,12 @@ static const char* const kDebugLayoutTransformation = "session.debug_layout_tran
 // Option values:
 // - "0": CPU EP fallback is not disabled. [DEFAULT]
 // - "1": CPU EP fallback is disabled.
-static const char* const kOrtSessionOptionsDisableCPUEPFallback = "session.disable_cpu_ep_fallback";
\ No newline at end of file
+static const char* const kOrtSessionOptionsDisableCPUEPFallback = "session.disable_cpu_ep_fallback";
+
+// Use this config when serializing a large model after optimization to specify an external initializers file
+static const char* const kOrtSessionOptionsOptimizedModelExternalInitializersFilePath =
+    "session.optimized_model_external_initializers_file_path";
```

Comment:
It is file name and not path (since it will save in same directory of output onnx file). Suggest to rename it to optimized_model_external_initializers_file_name

Reply:
Sure. I used path to be consistent with the original session option. https://github.com/microsoft/onnxruntime/blob/cf19c3697d126982acf45c040729269e53b5d9f4/onnxruntime/core/framework/session_options.h#L65